src/chart.zoom.js

```diff
@@ -198,7 +198,13 @@ function doZoom(chartInstance, zoomX, zoomY, center, whichAxes) {
 		chartInstance.update(0);
 
 		if (typeof zoomOptions.onZoom === 'function') {
-			zoomOptions.onZoom();
+			zoomOptions.onZoom({
+				chart: chartInstance,
+				zoomX: zoomX,
+				zoomY: zoomY,
+				center: center,
```

Comment:
I'm not sure `center` is a good name for this. It's probably better called `focalPoint`. See the discussion here https://github.com/chartjs/chartjs-plugin-zoom/pull/161#issuecomment-417947342. CC @nagix

Reply:
I've proposed to rename the argument in https://github.com/chartjs/chartjs-plugin-zoom/pull/192